src/c-opentimelineio/opentime-bindings/include/copentime/rationalTime.h

```diff
@@ -1,42 +1,57 @@
 #ifdef __cplusplus
-extern "C"{
+extern "C"
+{
 #endif
-#include <stdbool.h>
 #include "copentime/errorStatus.h"
-struct      RationalTime;
-typedef     struct RationalTime RationalTime;
+#include <stdbool.h>
```

Comment:
#includes shouldn't be contained within an extern "C" block, as it affects the linkage.

Reply:
Oh yes. I'll make the change. Negligence on my part.